internal/controllers/certmanager.go

```diff
@@ -68,6 +68,23 @@ func (r *Reconciler) setupTLS(ctx context.Context, cr *model.CryostatInstance) (
 		return nil, err
 	}
 
+	// Create a Cryostat CA certificate in each target namespace
+	for _, ns := range cr.TargetNamespaces {
+		cryostatCert := resources.NewCryostatCACert(cr, ns)
```

Comment:
Does this create a brand new certificate each time it's invoked or will it return a reference to an existing one? I think the idea here is that Cryostat has one cert of its own, and a copy of that same cert should be made available to other applications that may need it. If this creates a brand new (different) cert and places that into the other namespaces then I'm not sure the applications would be able to use that cert for what it's intended for.

Reply:
Right, we want to have one CA certificate per Cryostat. The workflow is like this:
1. Operator creates a cert-manager Certificate for the CA
2. cert-manager responds by creating a Secret containing the CA cert
3. Operator waits for this Secret and sets itself as the owner of it
4. New: Operator copies this Secret into each target namespace, and deletes the Secret from any removed target namespaces